packages/mdc-fab/_mixins.scss

```diff
@@ -0,0 +1,154 @@
+// Copyright 2016 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+@import "@material/animation/functions";
+@import "@material/animation/variables";
+@import "@material/elevation/mixins";
+@import "@material/ripple/mixins";
+@import "@material/theme/mixins";
+
+@function mdc-fab-theme-default-config_() {
+  @return (
+    standard: (
+      bg-color: secondary,
+      fg-color: text-primary-on-secondary,
+      ripple: (base-color: white, opacity: .16)
+    ),
+    plain: (
+      bg-color: white,
+      fg-color: text-primary-on-light,
+      ripple: ()
+    ),
+    disabled: (
+      bg-color: rgba(0, 0, 0, .12),
+      fg-color: rgba(0, 0, 0, .26),
+      ripple: null
+    ),
+  );
+}
+
+@function mdc-map-get-child_($parent-map, $keys...) {
+  $value: $parent-map;
+
+  @each $key in join($keys...) {
```

Comment:
Can you add a comment as to what this function does? The name and how it's used do kind of explain it, but when I first looked at the loop I expected that the reassignment and re-referencing of `$value` was a mistake.